cmd/thyra-server/main.go

```diff
@@ -14,6 +15,44 @@ import (
 	"github.com/massalabs/thyra/int/api/wallet"
 )
 
+func parseFlags(server *restapi.Server) {
+	flag.IntVar(&server.Port, "http-port", 80, "HTTP port to listen to")
+
+	flag.IntVar(&server.TLSPort, "https-port", 443, "HTTPS port to listen to")
+
+	certFilePtr := flag.String("tls-certificate", "", "path to certificate file")
+	keyFilePtr := flag.String("tls-key", "", "path to key file")
+	massaNodeServerPtr := flag.String("node-server", "", "IP address of the chosen massa node")
```

Comment:
```suggestion
	massaNodeServerPtr := flag.String("node-server", "", "Massa node that Thyra connects to. Can be an IP address, a URL or one of the following values: `TESTNET`, `LABNET` or `INNONET`.")
```

Why not setting INNONET as default value ?

Reply:
Indeed, in case the flag is empty INNONET will now be set by default.